### ISSUE TITLE
Crashes when changing TPS in 1.21

### ISSUE THREAD

**Issue by mhay10** [G4me4u/g4mespeed#13]
Crash Report: [https://pastebin.com/yMYY9p3g](https://pastebin.com/yMYY9p3g)

When ever I cut the TPS in half with the key assigned to the task, the game saves the world then crashses.

Error showed in launcher:
```
The game crashed whilst unexpected error
Error: java.lang.ClassCastException: class com.g4mesoft.module.tps.GSRenderTickCounterWrapper cannot be cast to class gg.essential.mixins.transformers.client.renderer.DynamicRenderTickCounterAccessor (com.g4mesoft.module.tps.GSRenderTickCounterWrapper and gg.essential.mixins.transformers.client.renderer.DynamicRenderTickCounterAccessor are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @69b794e2)
```

**Comment by G4me4u** [comment]
That should be easy enough to solve. I'll take a look at it at some point. It is a compatibility issue with another mod you have installed (essential).

**Comment by mhay10** [comment]
Removing essential worked, I would still like to use the essential mod in the future but it's more just because I'm used the UI it gives. I'll manage ;)

**Comment by G4me4u** [comment]
Just to give an update, this is fixed in v1.4.7-mc1.21.1.
You can download it from modrinth or curseforge.
